dev.yml

```diff
@@ -0,0 +1,35 @@
+# This file is for Shopify employees development environment.
+# If you are an external contributor you don't have to bother with it.
+name: shipit-engine
+
+up:
+  - homebrew:
+    - openssl
+    - sqlite
+  - ruby:
+      version: 2.1.8
+  - railgun
+  - bundler:
+      without: ci
+
+commands:
+  bootstrap: env REDIS_URL=redis://shipit-engine.railgun:6379 bin/bootstrap
+  console: pushd test/dummy; env REDIS_URL=redis://shipit-engine.railgun:6379 bin/rails console; popd
+  server: pushd test/dummy; env REDIS_URL=redis://shipit-engine.railgun:6379 bin/server -b 192.168.64.1 -p 55330; popd
+  test: env REDIS_URL=redis://shipit-engine.railgun:6379 bundle exec rake
+
+open:
+  shipit: https://shipit-engine.myshopify.io
```

Comment:
OS X tries to resolve `.localhost` via Bonjour, I think, and it can cause mysterious delays. It's a really sad state of affairs, or maybe I just don't understand what's happening. Give it a shot though, I guess if it works then :shipit: